package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "libp2p-gossipsub",
-  "version": "0.0.0",
-  "description": "An experimental Javascript implementation of gossip sub",
+  "version": "0.1.0",
```

Comment:
How do you feel about releasing `0.0.1` first?

I would like to have `0.1.0` with interop tests, but it is not a strong opinion